backend/internal/storage/postgres/schema/user.go

```diff
@@ -88,6 +88,30 @@ func (r *UserRepository) Follow(ctx context.Context, follower uuid.UUID, followi
 	if err != nil {
 		return false, err
 	}
+	println("inserted into follower table")
+
+	// get the username and profile picture of the followee
+	var followerUsername string
+	var followerProfilePicture string
+
+	// I tried to condesnse this into the insertion but im unsure about syntax,
+	// if its possible to do it in one query some help on that would be appreciated!
```

Comment:
you can add a `RETURNING` clause to the insert statement above! 

you can look at the UpdateReview method as an example
in `backend/internal/storage/postgres/schema/review.go`